packages/explat-client/package.json

```diff
@@ -21,5 +21,8 @@
 		"watch": "tsc --build ./tsconfig.json --watch",
 		"prepack": "yarn run clean && yarn run build",
 		"test": "yarn jest"
+	},
+	"dependencies": {
+		"@automattic/calypso-polyfills": "^1.0.0"
```

Comment:
Should this be a dev dependency since it's only used by tests? Or will it be used by non-test files?

Reply:
Ah yep it should be devDep, I thought I checked though 😮